### PR TITLE
Improve npm documentation for dependency management

### PR DESCRIPTION
**Pull request by Makena-Wambui**
Enhanced the description of npm's functionality by specifying that it installs, updates, and manages dependencies. Added an explanation of dependencies, noting that they are pre-written pieces of code such as libraries and packages essential for a Node.js application to function.

### CODE REVIEW

File: apps/site/pages/en/learn/getting-started/an-introduction-to-the-npm-package-manager.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ It started as a way to download and manage dependencies of Node.js packages, but
 
 ## Packages
 
-`npm` manages downloads of dependencies of your project.
+`npm` installs, updates and manages downloads of dependencies of your project. Dependencies are pre-written pieces of code such as libraries and packages that your Node.js application needs to work.
```
Comment (AugustinMauroy) [inline apps/site/pages/en/learn/getting-started/an-introduction-to-the-npm-package-manager.md]:
```suggestion
`npm` installs, updates and manages downloads of your project's dependencies. Dependencies are pre-built pieces of code, such as libraries and packages, that your Node.js application needs to work.
```

with a classier way of writing it

Comment (Makena-Wambui) [reply apps/site/pages/en/learn/getting-started/an-introduction-to-the-npm-package-manager.md]: Done, thank you.

File: apps/site/pages/en/learn/getting-started/an-introduction-to-the-npm-package-manager.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ It started as a way to download and manage dependencies of Node.js packages, but
 
 ## Packages
 
-`npm` manages downloads of dependencies of your project.
+`npm` installs, updates and manages downloads of dependencies of your project. Dependencies are pre-built pieces of code, such as libraries and packages, that your Node.js application needs to work.
```
Comment (ljharb) [inline apps/site/pages/en/learn/getting-started/an-introduction-to-the-npm-package-manager.md]:
```suggestion
`npm` installs, updates, and manages downloads of dependencies of your project. Dependencies are pre-built pieces of code, such as libraries and packages, that your Node.js application needs to work.  
```

File: apps/site/pages/en/learn/getting-started/nodejs-with-webassembly.md
Comment (AugustinMauroy) [inline apps/site/pages/en/learn/getting-started/nodejs-with-webassembly.md:1]: Please put these change in separated pr

Comment (Makena-Wambui) [reply apps/site/pages/en/learn/getting-started/nodejs-with-webassembly.md:1]: I have, thanks. Sorry for my mistake.

**Review by AugustinMauroy — APPROVED** [review]
LGMT !

**Review by bmuenzenmeyer — APPROVED** [review]
Thanks @Makena-Wambui !!

**Review by aymen94 — APPROVED** [review]

**Comment by aymen94** [comment]
@Makena-Wambui Congrats 😄, for you contribute.

**Comment by mikeesto** [comment]
Thanks @Makena-Wambui  . Can you run prettier to fix the formatting? Then we can merge this. Let me know if you have any problems

**Comment by Makena-Wambui** [comment]
@mikeesto I ran Prettier, and it confirmed that no changes were needed.

**Comment by bmuenzenmeyer** [comment]
this isn't your fault at all, there was a bug in our scripting - i resolved within https://github.com/nodejs/nodejs.org/commit/3d144f74e57067f3a966cc8794634130c8e61eae 

@Makena-Wambui i tried pushing to your fork but could not. you can replicate the fix, sync/rebase, or otherwise we can troubleshoot further

**Comment by Makena-Wambui** [comment]
@bmuenzenmeyer Please check if it is okay now.

File: apps/site/pages/en/learn/getting-started/an-introduction-to-the-npm-package-manager.md (hunk 2 above)
Comment (bmuenzenmeyer) [inline apps/site/pages/en/learn/getting-started/an-introduction-to-the-npm-package-manager.md]: let's try a manual fix from the UI here - there are still spaces at the end of the sentence.
```suggestion
`npm` installs, updates and manages downloads of dependencies of your project. Dependencies are pre-built pieces of code, such as libraries and packages, that your Node.js application needs to work.
```

**Review by bmuenzenmeyer — APPROVED** [review]

**Comment by bmuenzenmeyer** [comment]
yay - sorry this took so long @Makena-Wambui - it shouldn't be this hard (and that's on us)

**Comment by Makena-Wambui** [comment]
Yay. Thank you so much. My first contribution.

**Pull request merged**